components/nimbus/src/enrollment.rs

```diff
@@ -1015,6 +1037,7 @@ fn map_features(
     map
 }
 
+#[allow(unused)]
```

Comment:
I'm guessing that we'll need to start using this again once we start re-organizing `dbcache.rs`: this is the function that takes current enrollments and experiments and outputs the merged feature configs.

Reply:
Yep yep, just not being used now when the stateful feature is disabled.